docs/distributed/CONFIG.md

```diff
@@ -0,0 +1,65 @@
+## MinIO configuration YAML
+
+MinIO now supports starting the server arguments and configuration via a YAML configuration file. This YAML configuration describes everything that can be configured in a MinIO setup, such as '--address', '--console-address' and command line arguments for the MinIO server.
```

Comment:
I think better sentence could be `MinIO now supports starting the server using both command line arguments and configuration via a YAML configuration file` , just to be clear.

Reply:
Lets get @ravindk89's help in cleaning this up.

Reply:
Will merge the PR.